specs/_features/maxeb_increase/capella.py

```diff
@@ -3589,11 +3623,12 @@ def get_expected_withdrawals(state: BeaconState) -> Sequence[Withdrawal]:
             ))
             withdrawal_index += WithdrawalIndex(1)
         elif is_partially_withdrawable_validator(validator, balance):
+            excess = get_validator_excess_balance(validator, balance)
```

Comment:
consider renaming to `excess_balance` (as done in `is_partially_withdrawable_validator`)

Reply:
got rid of the temporary variable!